sdcm/utils/remote_logger.py

```diff
@@ -125,6 +125,17 @@ def _logger_cmd(self) -> str:
                '-u scylla-jmx.service'
 
 
+class SSHNonRootScyllaSystemdLogger(SSHLoggerBase):
+    """
```

Comment:
I think name should not contain the 'Systemd', because it use the tail.
better to call SSHNoneRootScyllaLogger or SSHNoneRootScyllaFileLogger

Reply:
The log still relates with systemd.

The log redirection is set in "$rsystemd"/scylla-server.service.d/nonroot.conf, it's scylla-server is controlled by systemd user.